extide/gradle/src/org/netbeans/modules/gradle/api/execute/RunUtils.java

```diff
@@ -347,41 +337,6 @@ public static ReplaceTokenProvider simpleReplaceTokenProvider(final String token
         };
     }
 
- /**
-     * Returns the active platform used by the project or null if the active
-     * project platform is broken.
-     * @param activePlatformId the name of platform used by Ant script or null
-     * for default platform.
-     * @return active {@link JavaPlatform} or null if the project's platform
-     * is broken
-     */
-    public static Pair<String, JavaPlatform> getActivePlatform(final String activePlatformId) {
```

Comment:
You can keep these two methods around, just `@deprecated`. Change the return type to `Pair` - e.g. the raw type. JVM is going to link to these two methods without a problem. See more in my [erasure](http://wiki.apidesign.org/wiki/Erasure) blog post.